WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -89,12 +101,25 @@ class GutenbergSettings {
         let blog = post.blog
 
         if post.isContentEmpty() {
-            return blog.editor.mobile == .gutenberg && blog.editor.web == .gutenberg
+            return shouldUseGutenbergForNewPosts(on: blog)
         } else {
             // It's an existing post
             return post.containsGutenbergBlocks()
         }
     }
+
+    private func shouldUseGutenbergForNewPosts(on blog: Blog) -> Bool {
```

Comment:
I missed this in 3e8bd9daaa9354ce4c7ddc9d1e33b30c9fa90f04 but this function does the exact same thing as `Blog.isGutenbergEnabled` right?

Reply:
True!
I like `blog.isGutenbergEnabled` :)

`shouldUseGutenbergForNewPosts` was created/renamed to follow up with Android.  `blog.isGutenbergEnabled` feels more Swifty ✨

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/pull/12189/commits/4fcabaf6beae7db9077647b36c8c2a10b9dc5564